runtime/compiler/env/J9CPU.cpp

```diff
@@ -28,8 +28,9 @@
 #include "env/CompilerEnv.hpp"
 #include "env/CPU.hpp"
 #include "env/VMJ9.h"
-#include "j9.h"
-#include "j9port.h"
+
+OMRProcessorDesc J9::CPU::_featureMasks = {OMR_PROCESSOR_UNDEFINED, OMR_PROCESSOR_UNDEFINED, {0, 0, 0, 0, 0}};
```

Comment:
In c++ initializing all elements wit 0 can be done with `array[10] = {0}` or even `array[10]={}`

Reply:
Note that statics are risky for multiple compilation threads and they usually don't work with JITServer